certified-connectors/Narvar/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1795 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0",
+    "title": "Narvar",
+    "description": "Narvar provides connection with Narvar portal. Alows create and update of orders on narvar portal as well as viewing already created orders.",
+    "contact": {
+      "name": "Microsoft Dynamics",
+      "url": "https://aka.ms/iomsupport",
+      "email": "iomdri@microsoft.com"
+    }
+  },
+  "host": "ws-st01.narvar.qa",
+  "basePath": "/api/v1/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/orders": {
+      "post": {
+        "summary": "POST - Create-Update Orders",
```

Comment:
@BekaAre all operation summaries should be a good sentence. please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations).
Please fix all operations' summaries in the swagger file.

Reply:
Thanks for pointing. Updated.